src/cli/cli.cpp

```diff
@@ -4267,7 +4266,7 @@ template <> otError Interpreter::Process<Cmd("partitionid")>(Arg aArgs[])
      * @sa otThreadGetPreferredLeaderPartitionId
      * @sa otThreadSetPreferredLeaderPartitionId
      * @par
-     * Note: OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE is required.
+     * 'OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE' is required.
```

Comment:
Make it backticks around OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE  so it's code font